### ISSUE TITLE
Possibility to skip columns

### ISSUE THREAD

**Issue by eborned** [unclead/yii2-multiple-input#20]
When I use the MultipleInput for a single field (for example, a list of BCC email addresses), the input models are posted as MyForm[bcc][bcc][0]. Would it be possible to skip the columns and just post it as MyForm[bcc][0]? This way, you can use the default 'each' validator with an existing rule. At the moment, a custom validation method seems to be required.

**Comment by eborned** [comment]
Never mind, the problem it with my model. The attribute in question is virtual from the ActiveRecord, and does not exist as a public attribute in my model. After adding the attribute as public value, it is recognized.

**Comment by eborned** [comment]
I'm reopening this. The problem I'm having seems to be in this line:

```
if (empty($this->columns) || (count($this->columns) == 1 && $this->model->hasProperty($name))) {
```

My ActiveRecord does not have a property 'cc', but it does have an attribute. After changing hasProperty to hasAttribute it works.

Would this be something that can be changed in upstream? Or is there a specific reason that hasProperty is used?

**Comment by unclead** [comment]
I think we can just add `$this->model->hasAttribute` to if statement

**Comment by unclead** [comment]
Fixed via bd990319289cac3c8457cc206c9cc5f6e9818499
@eborned please update to last revision and check that all is correct

**Comment by eborned** [comment]
It works, thanks!
